cinema/serializers.py

```diff
@@ -114,9 +114,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
```

Comment:
It is a good practice to add docstrings to your classes and methods. This will make your code more readable and maintainable. For example, you could explain the purpose of the `validate` method in the `TicketSerializer` class.